content/en/integrations/guide/oracle-check-upgrade-7.53.0.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Oracle Integration in the Agent 7.53.0
+kind: guide
+---
+
+# What has changed?
+
+- The Oracle integration is completely rewritten in the Agent release `7.53.0`.
+
+- The new check is easier to set up, and is less prone to configuration errors because it doesn't require any external clients such as Oracle JDBC drivers. The `jdbc` related parameters are ignored in the new check.
```

Comment:
"The new check is easier to set up, and is less prone to configuration errors because it doesn't require any external clients such as Oracle JDBC drivers." ➡️ "The new check doesn't require any external clients such as Oracle JDBC drivers. The `jdbc` related parameters are ignored in the new check."